src/control/property.rs

```diff
@@ -249,8 +249,14 @@ impl EnumValues {
     ///
     /// Note: This is a dumb translation, not every [`RawValue`] is part of en Enum
```

Comment:
Great, updated. I've now pretty much just moved the `unwrap` to `convert_value` which isn't desired for the user either. Before writing up a change, what do you prefer here?:

- Turn `Value::Enum(&'a EnumValue)` into `Value::Enum(Option<&'a EnumValue>)`;
- Return an `Option` directly from `convert_value`: this allows us to check the range for `(Un)SignedRange` too, and perhaps a few others, insert null checks etc?;
- Add an extra `Value::Invalid` or just return `Value::Unknown(RawValue)`?

Reply:
- `Value::Unknown` would be misleading, I guess, because it should be an Enum, we just failed to convert it.
- `Value::Invalid` also sounds weird, because usually the kernel should not send something invalid?
- I think the sensible approach would be just to do `Value::Enum(Option<&'a EnumValue>)`. I agree, that the `Option`-values could be factored out, but then the user would loose the information of the `Value`-type, which previously would have been returned by `convert_value`, but is now just `None`. I am not sure, how valuable this information really is, but if we want to simply/refactor this code, I would just do that in a separate issue/PR and move on with this.

Reply:
Fair enough. At that point we can also have a `Result<>` that could explicitly clarify what the failed-to-parse type is, and if it was ie. out of bounds, null, or not a valid enum value - all seem unlikely to happen.